test/suites/edgex-config-provider/main_test.go

```diff
@@ -59,8 +59,8 @@ func setup() (teardown func(), err error) {
 	}
 
 	// install the provider
-	if utils.LocalSnap() {
-		if err = utils.SnapInstallFromFile(nil, utils.LocalSnapPath); err != nil {
+	if utils.LocalServiceSnap() {
+		if err = utils.SnapInstallFromFile(nil, utils.LocalServiceSnapPath); err != nil {
```

Comment:
We could use the following variables:
`PLATFORM_SNAP` AND `SNAP` for path to snap file - matching the action input: `snap`.
`PLATFORM_CHANNEL` AND `CHANNEL` for snap channel when downloading from store - matching the action input: `channel. The `platform_channel` used only in config provider testing workflow can be removed.

For platform/no-sec tests:
- If PLATFORM_SNAP is not set, use SNAP as its value
- If PLATFORM_CHANNEL is not set, use CHANNEL as its value

To run a service test or config provider test and override both snaps platform and the testes snap, set `PLATFORM_SNAP=locally-build-edgex-go.snap SNAP=locally-built-app.snap`

To test the platform with a local snap: `SNAP=locally-build-edgex-go.snap` which would be equivalent to `PLATFORM_SNAP=locally-build-edgex-go.snap` because of the above conditional initializations.